requirements/dev.in

```diff
@@ -0,0 +1,3 @@
+-e git+https://github.com/kytos-ng/python-openflow.git#egg=python-openflow
+-e git+https://github.com/kytos-ng/sphinx-theme.git#egg=kytos-sphinx-theme
```

Comment:
```suggestion
```

Let's remove `kytos-sphinx-theme` it's not needed here.

Reply:
Unfortunately that will also imply having to regenerating requirements/dev.txt again

Reply:
Added `-e git+https://github.com/kytos-ng/kytos.git#egg=kytos` instead